x-pack/plugin/build.gradle

```diff
@@ -107,6 +107,9 @@ tasks.named("transformV7RestTests").configure({ task ->
   task.replaceKeyInDo("rollup.start_job", "xpack-rollup.start_job")
   task.replaceKeyInDo("rollup.stop_job", "xpack-rollup.stop_job")
   task.addAllowedWarningRegex(".*_xpack/rollup.* is deprecated.*")
+
+  task.replaceKeyInDo("ssl.certificates", "xpack-ssl.certificates")
+  task.addAllowedWarningRegex(".*_xpack/ssl.* is deprecated.*")
```

Comment:
I guess these are only used in `"Test get SSL certificates"` ? Maybe it would be worth applying them to that test only?
with `addAllowedWarningRegexForTest` and `replaceKeyInDo with a test name`

Reply:
👍, added the ability to do that in e2af78 and then used it in 7d254d8.